node/node.go

```diff
@@ -252,6 +252,12 @@ func (n *Node) StartConsensus(epoch uint32) error {
 	}
 
 	bootstrapper.StartSync()
+	epoch := uint32(0)
+	crtBlockHeader := n.blkc.GetCurrentBlockHeader()
+	if !check.IfNil(crtBlockHeader) {
+		epoch = crtBlockHeader.GetEpoch()
+		log.Debug("starting consensus", "epoch", epoch)
```

Comment:
I would put log.Info here, because it happens just once when you start the node

Reply:
done

Reply:
ok